Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1071,7 +1081,7 @@ async def listen(client: SocketModeClient, req: SocketModeRequest):
             if event.get('subtype') == 'bot_message':
                 demisto.debug("Received bot_message event type. Ignoring.")
                 return
-            if event.get('bot_id', None) in bot_id:
+            if event.get('bot_id', '') == bot_id:
```

Comment:
Simple fix for a `'in <string>' requires string as left operand, not NoneType` error.